tasks/lib/FileName.js

```diff
@@ -48,3 +55,21 @@ module.exports.toString = function(file) {
   string = string.replace(/%27/g,"'");
   return string;
 };
+/**
+* Return the next file in alphabetical order
+*/
+module.exports.nextFile = function(file) {
+  const files = fs.readdirSync(path.dirname(file))
+      .sort().filter(function(file) {
+        return path.extname(file) == '.html';
+      });
+  const pos = files.indexOf(path.basename(file));
+  if (pos == -1) {
+    return undefined;
+  }
+  //if it's the last one
+  if (pos == files.length - 1) {
```

Comment:
you don't need this in JS, if the element doesn't exist it already returns undefined.